modules/vulnerability-management20-configure-report-delivery-destinations-schedule.adoc

```diff
@@ -9,8 +9,23 @@
 You can optionally configure destinations for vulnerability reports and set up a schedule for delivery.
 
 .Procedure
-. To configure destinations for delivery, in the *Create report* window, under *Configure delivery destinations*, click *Add delivery destination*.
+. To configure destinations for delivery, in the *Create report* window, under *Configure delivery destinations*, you can add a delivery destination and set up a schedule for reporting.
++
+[NOTE]
+====
+If you chose to include CVEs that were discovered since the last scheduled report, you must configure a delivery destination and schedule. Otherwise, the delivery destination and schedule are optional.
+====
```

Comment:
I would make this note part of the abstract of this section instead, since previously it says "You can optionally configure..."